### PR TITLE
[Cookbook][WebServer] #5504 add a tip for the --force option

### PR DESCRIPTION
**Pull request by vincentaubert**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no |
| Applies to | 2.8 |
| Fixed tickets | #5504 |

### CODE REVIEW

File: cookbook/web_server/built_in.rst [hunk 1]
```diff
@@ -41,6 +41,15 @@ can change the socket passing an IP address and a port as a command-line argumen
 
 .. note::
 
+    You can use the ``--force`` option to force the web server start
+    even if a web server is already running.
```
Comment (xabbuh) [inline cookbook/web_server/built_in.rst]: I think we should reword this a bit. The issue with the current wording is that you may be tempted to try to use the option even when a web server instance is already running which in turn will lead to errors.

File: cookbook/web_server/built_in.rst [hunk 2]
```diff
@@ -41,6 +41,15 @@ can change the socket passing an IP address and a port as a command-line argumen
 
 .. note::
 
+    You can use the ``--force`` option to force the web server start
+    if the process wasn't correctly stopped (without using the server:stop command).
```
Comment (xabbuh) [inline cookbook/web_server/built_in.rst]: We should enclose `server:stop` with double backticks, but otherwise this looks really nice.

**Comment by xabbuh** [comment]
:+1:

**Comment by OskarStark** [comment]
@xabbuh why is this platfomsh check failing?

**Comment by WouterJ** [comment]
@OskarStark it's because these PRs don't contain the platform.sh configuration (as they were created before the configuration was added to the repository). When rebasing, they'll pass.

**Comment by WouterJ** [comment]
Thank you @vincentaubert for writing this nice and small note!

Fyi, I've added a versionadded box indicating that this option is new in Symfony 2.8 in 1739549.

**Comment by OskarStark** [comment]
thanks for the info @WouterJ

**Pull request merged**